src/applications/personalization/profile/components/direct-deposit/alerts/DirectDepositBlocked.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+export default function DirectDepositBlocked() {
+  return (
+    <va-alert status="warning" visible>
+      <h2
+        slot="headline"
+        className="vads-u-font-size--base vads-u-font-family--sans"
+        data-testid="direct-deposit-blocked"
+      >
+        We can’t show your financial information
+      </h2>
+      <p>
+        We’re sorry. Based on our records, we can't show your financial
+        information in your VA.gov profile.
+      </p>
+      <p>
+        If you think this is an error, you can call the VA.gov help desk at{' '}
+        <va-telephone contact="855-574-7286" /> (
+        <a href="tel:711" aria-label="TTY: 7 1 1.">
```

Comment:
It should be, the `va-telephone` doesn't allow what the mockups need. The requirements need `TTY: 711` to be the link, but the number to only be `711`. 

If you know a way to do that that would be awesome!

Reply:
I see, I missed the screenshot in the description. For what you are needing to do there, an `a` tag is your best bet

Reply:
This is interesting. I wonder if we should talk to the design systems team on this to make a universally accessible TTY link component (or add functionality to existing va-telephone component), since it looks like it is pretty widely used and the va-telephone component doesn't allow for the customization needed.